lib/temporal/SQL/update_stds_spatial_temporal_extent_template.sql

```diff
@@ -14,53 +15,58 @@
     		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_relative_time SET start_time =
-       (SELECT min(start_time) FROM GRASS_MAP_relative_time WHERE GRASS_MAP_relative_time.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_relative_time SET end_time =
-       (SELECT max(end_time) FROM GRASS_MAP_relative_time WHERE GRASS_MAP_relative_time.id IN
+       ) AS new_stats
+ WHERE STDS_absolute_time.id = 'SPACETIME_ID';
+
+UPDATE STDS_relative_time
+   SET
+       STDS_relative_time.start_time = new_stats.start_time_new,
+       STDS_relative_time.end_time = new_stats.end_time_new
+  FROM
+       (SELECT
+           min(start_time) AS start_time_new,
+           max(end_time) AS end_time_new
+       FROM GRASS_MAP_relative_time WHERE GRASS_MAP_relative_time.id IN
     		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
+       ) AS new_stats
+ WHERE STDS_relative_time.id = 'SPACETIME_ID';
+
 -- Update the spatial extent
-UPDATE STDS_spatial_extent SET north =
-       (SELECT max(north) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET south =
-       (SELECT min(south) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET east =
-       (SELECT max(east) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET west =
-       (SELECT min(west) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET top =
-       (SELECT max(top) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET bottom =
-       (SELECT min(bottom) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
-    		(SELECT id FROM SPACETIME_REGISTER_TABLE)
-       ) WHERE id = 'SPACETIME_ID';
-UPDATE STDS_spatial_extent SET proj =
-       (SELECT min(proj) FROM GRASS_MAP_spatial_extent WHERE GRASS_MAP_spatial_extent.id IN
+UPDATE STDS_spatial_extent
+   SET
+       STDS_spatial_extent.north = new_stats.north_new,
+       STDS_spatial_extent.south = new_stats.south_new,
+       STDS_spatial_extent.east = new_stats.east_new,
+       STDS_spatial_extent.west = new_stats.west_new,
+       STDS_spatial_extent.top = new_stats.top_new,
+       STDS_spatial_extent.bottom = new_stats.bottom_new,
+       STDS_spatial_extent.proj = new_stats.proj_new
+       (SELECT
```

Comment:
Is it missing a `FROM` or there is an alternative syntax?

Reply:
Thanks. Good catch. Fixed.